hg2git.py

```diff
@@ -138,3 +137,8 @@ def get_git_sha1(name,type='heads'):
     return l[0:40]
   except subprocess.CalledProcessError:
     return None
+
+def change_encoding(name,encoding)
```

Comment:
Call it `force_utf8(name,encoding)` to make it clear what it does

Reply:
Fixed

Reply:
Please let me resolve the conversation when I'm happy with your fix of the comment. That way I can ignore old stuff, and won't have to go hunting through all resolved comments just to find out what has changed.